experimental/PropertyDDS/packages/property-shared-tree-interop/src/test/tableExampleSchemaConverter.spec.ts

```diff
@@ -0,0 +1,211 @@
+		checkInheritanceTranslation(schemaRepository);
+	});
+});
+
+function register() {
+	PropertyFactory.register({
+		typeid: "Test:Cell-1.0.0",
+		properties: [{ id: "value", typeid: "Uint64" }],
+	});
+
+	PropertyFactory.register({
+		typeid: "Test:RowProperty-1.0.0",
+		properties: [{ id: "value", typeid: "String" }],
+	});
+
+	PropertyFactory.register({
+		typeid: "Test:RowInfo-1.0.0",
+		properties: [{ id: "value", typeid: "Uint64" }],
+	});
+
+	PropertyFactory.register({
+		typeid: "Test:Row-1.0.0",
+		properties: [{ id: "cells", typeid: "Test:Cell-1.0.0", context: "array" }],
+	});
+
+	PropertyFactory.register({
+		typeid: "Test:ExtendedRow-1.0.0",
+		inherits: ["Test:Row-1.0.0"],
+		properties: [
+			{ id: "info", typeid: "Test:RowInfo-1.0.0", context: "map" },
+			{ id: "props", typeid: "Test:RowProperty-1.0.0", context: "map" },
+		],
+	});
+
+	PropertyFactory.register({
+		typeid: "Test:OtherExtendedRow-1.0.0",
+		inherits: ["Test:Row-1.0.0"],
+		properties: [
+			{ id: "info", typeid: "Test:RowInfo-1.0.0", context: "map" },
+			{ id: "props", typeid: "Test:RowProperty-1.0.0", context: "map" },
+		],
+	});
+
+	PropertyFactory.register({
+		typeid: "Test:Table-1.0.0",
+		properties: [
+			{ id: "rows", typeid: "Test:Row-1.0.0", context: "array" },
+			{ id: "extendedRows", typeid: "Test:ExtendedRow-1.0.0", context: "array" },
+			{
+				id: "encoding",
+				typeid: "Enum",
+				properties: [
+					{
+						id: "none",
+						value: 1,
+					},
+					{
+						id: "utf8",
+						value: 2,
+					},
+					{
+						id: "base64",
+						value: 3,
+					},
+				],
+			},
+		],
+	});
+
+	PropertyFactory.register({
+		typeid: "Test:DescribedTable-1.0.0",
+		inherits: ["Test:Table-1.0.0"],
+		properties: [{ id: "description", typeid: "String" }],
+	});
+}
+
+function checkMissingRefs(schemaData) {
+	const schemaMap = schemaData.treeSchema;
+	const schemaTypesSet = new Set<string>();
+	let keysIter = schemaMap.keys();
+	for (const key of keysIter) {
+		schemaTypesSet.add(key);
+	}
+	keysIter = schemaMap.keys();
+	for (const key of keysIter) {
+		const value = schemaMap.get(key);
+		value?.localFields.forEach((field) => {
+			if (field.types) {
+				field.types.forEach((type) => {
+					if (!schemaTypesSet.has(type.toString())) {
+						fail(
+							`Missing type ${type.toString()} in schema at the type ${key} field ${
+								field.name
+							}`,
+						);
+					}
+				});
+			}
+		});
+	}
+}
+
+function checkInheritanceTranslation(schemaData) {
+	const schemaMap = schemaData.treeSchema;
+	const row = schemaMap.get("array<Test:Row-1.0.0>");
+	expect(row).not.toBeUndefined;
+	expect(row?.localFields).not.toBeUndefined;
+	const field = row?.localFields.get("");
+	expect(field).not.toBeUndefined;
+	expect(field?.types).not.toBeUndefined;
+	const types = field?.types;
+	expect(types?.has("Test:Row-1.0.0")).toBeTruthy;
+	expect(types?.has("Test:ExtendedRow-1.0.0")).toBeTruthy;
+	expect(types?.has("Test:OtherExtendedRow-1.0.0")).toBeTruthy;
```

Comment:
As I mentioned above, skipping `()` here provided false-positive results, as the code which should fetch inheriting child types for arrays is missing. Please, check my PR.